core/controllers/editor_test.py

```diff
@@ -2587,7 +2587,7 @@ def test_draft_updated_version_valid(self):
         self.assertTrue(response['is_version_of_draft_valid'])
         self.assertEqual(response['draft_change_list_id'], 2)
 
-    def test_draft_updated_version_invalid(self):
+    def test_draft_does_not_updated_version_invalid(self):
```

Comment:
This function name looks a bit odd. Can you rephrase it so it is more readable?

Reply:
Changed it.